spaces/debaseonomics/index.json

```
@@ -0,0 +1,21 @@
+{
+	"name": "Debaseonomics",
+	"network": "1",
+	"symbol": "DEGOV",
+	"skin": "your-space",
```

The css selector should similar to the space directory name.
```suggestion
	"skin": "debaseonomics",
```